src/instance.rs

```diff
@@ -21,6 +21,15 @@ pub unsafe trait PyNativeType: Sized {
     fn py(&self) -> Python {
         unsafe { Python::assume_gil_acquired() }
     }
+    /// Cast `&PyAny` to `&Self` without no type checking.
+    ///
+    /// # Safety
+    ///
+    /// Unless obj is not an instance of a type corresponding to Self,
```

Comment:
You have a accidental double negative.
Did you mean: 
```rust
/// Unless `obj` is an instance of a type corresponding to `Self`,
```

Reply:
Thanks!